src/wmtk/utils/polygon_mesh_topology_initialization.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <wmtk/Types.hpp>
```

Comment:
you're probably lucking out on includes thusfar, but you need to include vector/tuple for this to really "work" (if someone re-orders some headres your code will fail to compile)